packages/cli/src/Doctor.ts

```diff
@@ -77,7 +77,7 @@ ${chalk.bold('Examples')}
 
     const schema = await readFile(schemaPath, 'utf-8')
     const localDmmf = await getDMMF({ datamodel: schema })
-    const config = await getConfig({ datamodel: schema })
+    const config = await getConfig({ datamodel: schema, ignoreEnvVarErrors: false })
```

Comment:
I confirm `ignoreEnvVarErrors` was set as `false` by default in Rust:

https://github.com/prisma/prisma-engines/blob/84e58e9dcfa1502d66bbdefced378d9a41dd1d57/prisma-fmt/src/get_config.rs#L6-L11